README.md

```diff
@@ -1 +1,105 @@
+
+# Audience
+
+IPVM is intended for general distributed computation, but excels at tasks that would be found in systems like Lambda Step functions. The IPVM network itself is focused on machine-to-machine interaction — where each node acts as the user's agent — but is not expected to be interacted with directly when on the happy path.
+
+# Goals
+
+IPVM means to provide:
+
+- Task matchmaking network / marketplace
+- Pluggable execution engine
+- A trust layer (e.g. verifiable compute, encryption, PKI, capabilities)
+- Seamlessly tie together external services
+- Interfaces open to extension
+- First-class P2P payments
+
+IPVM is application agnostic, but intends to abstract concerns of locality away from the programmer.
+
+## Antigoals
+
+IPVM will _not_ attempt to develop into a:
+
+- replacement or upgrade of IPFS internals
+- high level language for distributed applications
+- custom Wasm runtime
+- blockchain or other global consensus mechanism
+
+IPVM's focus is on execution, networking, data flow, and trust. It is important to make the internals debuggable and friendly where possible, but only when it doesn't conflict with the core aim at this layer: efficiency and abstracting away distributed systems concerns.
+
+IPVM's primary focus is on machines and efficiency. Human-friendliness is important, but many human-oriented features should be pushed into higher layers. These may include high-level APIs, language integration, etc.
+
+# Design Principles
+
+> You can prove anything you want by coldly logical reason — if you pick the proper postulates.
+> 
+> — Asimov, "I, Robot"
+
+1. Proliferate!
+2. Open world / extension without prenegotiation
+3. Participatory: anyone can consume, serve, or both
+4. Learn into our strengths: IPVM is not k8s, so don't try to be
+5. Common things should be simple, and complex things should be possible
+6. Handle low-level details (networking, failure, trust) so that others can focus on business logic
+7. Build in layers with clear audiences (machine-to-machine, end-user, etc)
+
+When in doubt, optimize for proliferation. Get implementations into many hands, find services that already tie-in at the RPC and trust layers ([UCAN]), and avoid having the core IPVM team become a bottleneck for extending the network. [Richard Gabriel] described Unix and C as being like (helpful) computer viruses due to how they spread and integrate with other systems.
```

Comment:
minor: cite gabriel's worse is better (I think that's where the description come from)

Reply:
It's from his "Models of Software Acceptance: How Winners Win" — good point, will update :)